index.html

```diff
@@ -705,31 +710,68 @@
                                 if (cellvar.model.in.height == null) {
                                     inputPort = 0;
-                                } else {
+                                }
+                                else {
                                     inputPort = cellvar.model.in.height;
                                 }
                                 if (cellvar.model.out.height == null) {
                                     outputPort = 0;
-                                } else {
+                                }
+                                else {
                                     outputPort = cellvar.model.out.height;
                                 }
                                 if (cellvar.model.evtin.height == null) {
                                     controlPort = 0;
-                                } else {
+                                }
+                                else {
                                     controlPort = cellvar.model.evtin.height;
                                 }
                                 if (cellvar.model.evtout.height == null) {
                                     commandPort = 0;
-                                } else {
+                                }
+                                else {
                                     commandPort = cellvar.model.evtout.height;
                                 }
+
+                                var flip = false, mirror = false;
+                                var style = cell.style;
+                                var styleObject = styleToObject(style);
+                                if(styleObject['stencilFlipV'] == null){
+                                    flip = false;
+                                }
+                                else
+                                {
+                                    
+                                    if(styleObject['stencilFlipV'] == 0){
+                                        flip = false;
+                                    }
+                                    else{
+                                        flip = true;
+                                    }
+
+                                }
+                                if(styleObject['stencilFlipH'] == null){
+                                    mirror = false;
+                                }
+                                else
+                                {
+                                   
+                                    if(styleObject['stencilFlipH'] == 0 ){
+                                        mirror = false;
+                                    }
+                                    else{
+                                        mirror = true;
+                                    }
+
+                                }
+                                style = objectToStyle(styleObject);
```

Comment:
Oh nevermind. The control will not go inside the if condition when there's already an ';' present at the end of the string because of objectToStyle function. Looks good!